lighthouse-core/test/fixtures/source-maps/angular.min.js

```diff
@@ -0,0 +1,316 @@
+/*
```

Comment:
this will need to go under `third-party/angular/` (see https://opensource.google/docs/releasing/preparing/#third-party-components for how it works), but is there a smaller dependency we could add that's real js and source map but not 1.5MB instead? We could e.g. compile some of our own current code with a source map and it could exist in `test/fixtures/` without worrying about licenses.

Reply:
actually yeah we don't need this - I'll remove it